test/bases/renku_data_services/data_api/test_sessions.py

```diff
@@ -390,6 +392,27 @@ async def test_get_project_launchers(
     assert {launcher["name"] for launcher in launchers} == {"Launcher 2", "Launcher 3"}
 
 
+def test_env_variable_validation():
+    renku_name_env_variables = {
+        "RENKU_KEY_NUMBER_1": "a value",
+        "RENKULAB_THING": "another value",
+    }
+    try:
+        EnvVar.from_dict(renku_name_env_variables)
+        assert False  # should not reach this point
+    except errors.ValidationError as e:
+        assert e.message == "Env variable name 'RENKU_KEY_NUMBER_1' should not start with 'RENKU'."
```

Comment:
Pytest can help you handle expected assertions without using try/except. Sorry I missed this on the first round.

https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
Fixed.